gulpfile.js/tasks/html.js

```diff
@@ -24,14 +24,17 @@ var htmlTask = function() {
     return JSON.parse(fs.readFileSync(dataPath, 'utf8'))
   }
 
+  var customFilters = TASK_CONFIG.html.customFilters || function(env) {}
```

Comment:
I'll test that quickly. I was also thinking the fallback would show that available `env` var but that's better handled by the docs like you pointed out.